core/modules/book/book.install

```diff
@@ -138,6 +141,18 @@ function book_update_1002() {
   $config->save();
 }
 
+/**
+ * Creates the table to enable caching of Book navigation.
+ */
+function book_update_1003() {
+  $table = backdrop_get_schema_unprocessed('system', 'cache');
+  $table['description'] = "Cache table for Book to store book navigation data.";
+  if (db_table_exists('cache_book')) {
+    db_drop_table('cache_book');
+  }
+  db_create_table('cache_book', $table);
```

Comment:
If you have the contrib `book_cache` installed, then run this hook, and then uninstall `book_cache`, the table `cache_book` gets deleted.

I realize that there is no official release of `book_cache`, but who knows how many installations of the dev version there are. It'd be safer to do something similar to `system_update_1098()`, where the hook disables the contrib module if found by changing the `system` table, instead of uninstalling it (which drops the table). 

Additionally, it'd be a good idea to add `book_cache` to the list of modules incorporated to core in `backdrop_merged_modules()`